akka-actor-typed-tests/src/test/scala/docs/akka/typed/FaultToleranceDocSpec.scala

```diff
@@ -8,7 +8,9 @@ import akka.actor.typed.scaladsl.Behaviors
 import akka.actor.typed.{ DeathPactException, SupervisorStrategy }
 import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
 import org.scalatest.WordSpecLike
+import com.github.ghik.silencer.silent
 
+@silent
```

Comment:
add a pattern? Or is it multiple

Reply:
Yes, there are two instances where `context` would need to be `_` for compiler joy, so since this is for sample usage and useful to keep that there (again, sorry @raboof) https://github.com/akka/akka/pull/27433